js/src/block/edit.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import Edit from './edit';
+
+// Mock the <InpectorControls> component only, so that the other components in this package behave as usual.
+jest.mock( '@wordpress/block-editor', () => {
+	const original = require.requireActual( '@wordpress/block-editor' );
```

Comment:
This mocks `InspectorControls`, which is necessary, as it's a Slot/Fill component. Without this, its children wouldn't render.

But it's not possible to mock the entire `@wordpress/block-editor` package, as `MediaPlaceholder` is in that. 

A simple `jest.mock( '@wordpress/block-editor' )` would mock that entire package, and the `MediaPlaceholder` wouldn't render.